src/sentry/static/sentry/app/components/sidebar/serviceIncidents.tsx

```diff
@@ -111,36 +115,8 @@ class ServiceIncidents extends React.Component<Props, State> {
 
 export default ServiceIncidents;
 
-const IncidentList = styled('ul')`
-  font-size: 13px;
-  list-style: none;
-  padding: ${space(3)} ${space(3)} 0;
-`;
-
-const IncidentItem = styled('li')`
-  border-bottom: 1px solid ${p => p.theme.innerBorder};
-  margin-bottom: ${space(3)};
-  padding-bottom: ${space(0.75)};
-`;
+const IncidentList = styled('div')``;
```

Comment:
am not an expert but isn't this just `<div>`?

Reply:
I left this alone because the tests were referencing IncidentList and I didn't feel like rewriting that